src/inspector/inspectorwebsocketconnection.cpp

```diff
@@ -512,6 +521,13 @@ static QList<WebSocketCommand> s_commands{
                        return obj;
                      }},
 
+    WebSocketCommand{"translate", "Translate a string", 1,
```

Comment:
This is not used, yet. Right? At least in the tests.

Reply:
No, there's nothing that uses it (yet).

I was finding that the tests would fail if my locale was set to anything other than `en`, so I had the idea that maybe we could compare a results against a translated string. But it turned out to be easier to just set `LANGUAGE=en` at the start of the test scripts.